styles.css

```diff
@@ -70,205 +283,160 @@ body {
-    gap: 50px;
-    padding-left: 16px;
-    padding-right: 16px;
-  }
-
-  .Tonic {
-    color: #172b4d;
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 32px;
-    line-height: 44px;
+  .card1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 16px;
+    border-radius: 16px;
+    border: solid #e5e5e5 1px;
+    background-color: #fff;
   }
 
-  .canopy {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    gap: 12px;
+  .about {
+    background-color: #fff;
+    border-radius: 0 100px 0 0 / 100px;
+    padding-top: 8%;
+    margin-top: 16%;
   }
 
-  .canopi {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #344563;
+  .aboutmyself {
+    margin-left: 8vw;
+    margin-right: 8vw;
   }
 
-  .backend {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #7a869a;
+  .contactform {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+    gap: 50px;
+    background: #6070ff;
+    background-image: url(icons/contact-form-shapes-mobile@2x.png);
+    background-size: cover;
+    border-radius: 100px 0 0 0 / 100px;
   }
 
-  .fiftin {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #7a869a;
+  .form {
+    display: flex;
+    flex-direction: column;
   }
 
-  .art {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 15px;
-    line-height: 24px;
-    color: #344563;
+  #name {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    color: #172b4d;
+    width: 80vw;
+    height: 5vh;
+    margin-left: 8vw;
   }
 
-  .frontend {
-    font-family: 'Poppins', Arial, Helvetica, sans-serif;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    color: #6070ff;
-    list-style-type: none;
-    display: flex;
-    gap: 16px;
-    padding-left: 0;
+  #email {
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    color: #172b4d;
+    width: 80vw;
+    height: 5vh;
+    margin-left: 8vw;
   }
 
-  .frontend li {
-    background: #ebebff;
-    padding: 4px 12px;
+  #textarea {
+    background: #fff;
+    border: 1px solid #cfd8dc;
     border-radius: 8px;
+    color: #172b4d;
+    width: 80vw;
+    height: 25vh;
+    margin-left: 8vw;
   }
 
-  .seeproject {
+  .submitted {
+    cursor: pointer;
+    padding: 12px 16px;
+    width: 160px;
+    height: 48px;
+    border: 1px solid #cfd8dc;
+    background: #fff;
+    border-radius: 8px;
+    color: #6070ff;
+    letter-spacing: 0.03em;
```

Comment:
> - [ ]  I noticed that the hover and pressed button state interactions are missing from your button. As this is a project requirement, Please kindly ensure to implement them. This will improve the overall user experience for anyone who uses your website.😊
> 
> Figma design👇:
> 
> ![des-figma](https://user-images.githubusercontent.com/40009354/236897761-bffcfff4-35f7-4e91-a3e2-5d7c24d77d78.png)
> 
> Your design👇:
> 
> ![des](https://user-images.githubusercontent.com/40009354/236897886-cd7b00d2-9587-479e-9432-38155bed5676.png)